src/site/stages/prearchive/link-assets-to-bucket.js

```diff
@@ -6,15 +6,57 @@
  */
 
 /* eslint-disable no-continue */
+/* eslint-disable no-console */
 
 const fs = require('fs');
-const jsdom = require('jsdom');
 const path = require('path');
+const cheerio = require('cheerio');
 const buckets = require('../../constants/buckets');
 
 const TEAMSITE_ASSETS = 'va_files';
 
+function getAssetLinkElements(doc) {
+  const elementTags = 'script, img, link, picture > source';
```

Comment:
This should probably be defined outside of the scope of this function so it doesn't get instantiated each time. And in the case that it is a constant in a wider scope, this function might not need to exist either.

```
const ASSET_LINK_TAGS = 'script, img, link, picture > source';

...

const doc = cheerio.load(htmlFile);
const assetLinkElements = doc(ASSET_LINK_TAGS);
```

Reply:
Good point, thanks!